src/java.base/share/classes/sun/security/ssl/X509KeyManagerImpl.java

```diff
@@ -614,7 +614,7 @@ CheckResult check(X509Certificate cert, Date date,
                             // exchange and not ephemeral ECDH. We leave it in
                             // for now until there are signs that this check
                             // causes problems for real world EC certificates.
-                            if ((this == SERVER) && (getBit(ku, 4) == false)) {
+                            if ((this == SERVER) && (!getBit(ku, 4))) {
```

Comment:
You could remove the inner parenthesis around `this == SERVER` and `!getBit(ku, 4)`, which would make it consistent with the style used on line 584.